File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        StringBuilder sb = new StringBuilder();
        sb.append(my_string.substring(0,s));
        sb.append(overwrite_string);
        sb.append(my_string.substring(s+overwrite_string.length(), my_string.length()));
        String answer = sb.toString();
        return answer;
    }
}

File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.go
func solution(my_string string, overwrite_string string, s int) string {
    	s1 := my_string[:s]
	s2 := my_string[s+len(overwrite_string):]
	runes := []rune(s1)
	for _, v := range overwrite_string {
		runes = append(runes, v)
	}
	for _, v := range s2 {
		runes = append(runes, v)
	}
	result := string(runes)
	return result
}